components/TextFields/src/MDCTextField.m

```diff
@@ -58,6 +58,8 @@ @interface MDCTextField () {
 @implementation MDCTextField
 
 @dynamic borderStyle;
+@synthesize mdc_elevationDidChangeBlock = _mdc_elevationDidChangeBlock;
+@synthesize mdc_overrideBaseElevation = _mdc_overrideBaseElevation;
```

Comment:
Do we not need to add these properties to the header file?

Reply:
They come from the protocol.